### DISCUSSION TITLE
Black background after converting Tiff to PDF

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#1210] [Help]
Today  I upgraded Magick.NET to a new version and the attached Tiff file started to have a black background when converted to a PDF file.  It was fine before the upgrade. 

[test.zip](https://github.com/dlemstra/Magick.NET/files/9019245/test.zip)

**Comment by dlemstra** [comment]
Can you define what fine means? And with a new version you are talking about the latest version?

**Comment by tomasr78** [comment]
This is how Tiff appears

![image](https://user-images.githubusercontent.com/2582291/176756070-6e8d21a6-7b3e-44b0-94b2-322c75b35886.png)

This is how PDF was generated with the Magick.NET version

```
  <package id="Magick.NET-Q8-OpenMP-x64" version="11.1.2" targetFramework="net47" />
  <package id="Magick.NET.Core" version="11.1.2" targetFramework="net47" />
```

![image](https://user-images.githubusercontent.com/2582291/176756370-797f1172-5aeb-4a9d-9048-ab52d58b6942.png)

This is now after installing the latest version
```

  <package id="Magick.NET-Q8-OpenMP-x64" version="11.2.1" targetFramework="net47" />
  <package id="Magick.NET.Core" version="11.2.0" targetFramework="net47" />
```

![image](https://user-images.githubusercontent.com/2582291/176756497-bf6893d1-af25-453b-af26-e8884a552b36.png)

**Comment by dlemstra** [comment]
Thanks for all the info and including those version numbers made it really easy to figure out what caused this. Now I will need to figure out how we can resolve this inside ImageMagick.

**Reply by tomasr78** [reply]
Any update on this? Should we revert to the old version until the issue will be fixed in ImageMagick?

**Comment by dlemstra** [comment]
This will be resolved in the next release.

**Reply by tomasr78** [reply]
Do you know ETA?

**Comment by dlemstra** [comment]
Since you are a sponsor I could probably publish a release later today if that would help you.

**Reply by tomasr78** [reply]
That would be great! Thank you!
